### PR TITLE
Stellar: getPaymentDetailsLocal changes for chat

### PR DESCRIPTION
**Pull request by zapu**
- Change `getPaymentDetailsLocal` to accept nullable `accountID` parameter. Chat previews do not know account IDs, so if they want unformatted message (because service will not know whether it's a debit or credit without knowing `accountID`), they can pass `nil` as `accountID`.
- Add `TxID` to `SendPaymentResLocal`. No reason not to, service already had it, and it helped me with debugging. I'll also be fine without that change if anyone objects.
- Add `TestPaymentDetailsEmptyAccId` test that simulates how chat would pull payment details with nil `accountID`.

### CODE REVIEW

**Comment by zapu** [comment]
cc @buoyad

File: go/stellar/stellarsvc/frontend.go [hunk 1]
```diff
@@ -1282,6 +1288,7 @@ func (s *Server) SendPaymentLocal(ctx context.Context, arg stellar1.SendPaymentL
 	return stellar1.SendPaymentResLocal{
 		KbTxID:  sendRes.KbTxID,
 		Pending: sendRes.Pending,
+		TxID:    sendRes.TxID,
```
Comment (patrickxb) [inline go/stellar/stellarsvc/frontend.go]: @buoyad do you want this?

Comment (zapu) [reply go/stellar/stellarsvc/frontend.go]: I can remove this. I needed it for debugging one time while working on tests for this PR.

Comment (patrickxb) [reply go/stellar/stellarsvc/frontend.go]: If you want to keep it, how about returning a PaymentID?

Comment (buoyad) [reply go/stellar/stellarsvc/frontend.go]: Frontend only uses PaymentID

Comment (patrickxb) [reply go/stellar/stellarsvc/frontend.go]: Ok, @zapu can you either remove or change to PaymentID?  thanks.

Comment (zapu) [reply go/stellar/stellarsvc/frontend.go]: will do, thanks!
